airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -760,7 +760,7 @@
   sourceDefinitionId: eff3616a-f9c3-11eb-9a03-0242ac130003
   dockerRepository: airbyte/source-google-analytics-v4
   dockerImageTag: 0.1.34
-  documentationUrl: https://docs.airbyte.com/integrations/sources/google-analytics-universal-analytics
+  documentationUrl: https://docs.airbyte.com/integrations/sources/google-analytics-data-api
```

Comment:
Shouldn't this match the docker repository? This explicitly matches the other docker repository/source module

Reply:
see 
https://github.com/airbytehq/airbyte/blob/5512befeb1ac2c1d3348dd20df0ed31b11e1c4d3/docs/integrations/sources/google-analytics-data-api.md

vs 

https://github.com/airbytehq/airbyte/blob/5512befeb1ac2c1d3348dd20df0ed31b11e1c4d3/docs/integrations/sources/google-analytics-v4.md

google-analytics-v4.md contains the docs for universal analytics/source-google-analytics-v4

Reply:
This also follows what Alex said: 
> Changes needed:
Google Analytics (UA)
connector definition id: eff3616a-f9c3-11eb-9a03-0242ac130003
documentation url: https://docs.airbyte.com/integrations/sources/google-analytics-v4/
Google Analytics 4
connector definition id: 3cc2eafd-84aa-4dca-93af-322d9dfeec1a 
documentation url: https://docs.airbyte.com/integrations/sources/google-analytics-data-api/

Reply:
+ URL on line 771 should be `https://docs.airbyte.com/integrations/sources/google-analytics-data-api/`

Reply:
OH! Thank you, I read that wrong.
... so the v4 connector doesn't get the "V4" docs.  Got it :D

Reply:
The GA4 connector doesn't, yeah - the v4 one is universal analytics... for some cursed reason 😅